master/buildbot/scripts/start.py

```diff
@@ -56,6 +57,10 @@ def _failure(self, why):
 file during startup. Please inspect and fix master.cfg, then restart the
 buildmaster.
 """)
+        elif why.check(BuildmasterStartupError):
+            print("""
+The buildmaster startup failed. Please see 'twistd.log' for possible reason.
```

Comment:
Well we can first do that on the code that we rework, as we do for inlineCallbak